docker/kubernetes-tentacle/Dockerfile

```diff
@@ -46,9 +46,10 @@ ENV TargetTenant=""
 ENV TargetTenantTag=""
 ENV TargetTenantedDeploymentParticipation=""
 ENV OCTOPUS__K8STENTACLE__NAMESPACE=""
-ENV OCTOPUS__K8STENTACLE__USEJOBS="True"
-ENV OCTOPUS__K8STENTACLE__JOBSERVICEACCOUNTNAME=""
-ENV OCTOPUS__K8STENTACLE__JOBVOLUMEYAML=""
+ENV OCTOPUS__K8STENTACLE__EXECUTEINLOCALSHELL="False"
```

Comment:
@flin-8 and I discussed the old `USEJOBS` and have inverted this flag so that it's _not_ the local shell, it's just _something else_